packages/modules/providers/auth-google/package.json

```diff
@@ -17,10 +17,9 @@
   "author": "Medusa",
   "license": "MIT",
   "scripts": {
-    "prepublishOnly": "cross-env NODE_ENV=production tsc --build",
     "test": "jest --passWithNoTests src",
     "test:integration": "jest --forceExit -- integration-tests/**/__tests__/**/*.spec.ts",
-    "build": "rimraf dist && tsc -p ./tsconfig.json",
+    "build": "rimraf dist && tsc --build ./tsconfig.json",
```

Comment:
Suggestion : same where it is specified and not different that the default config file name